.github/integration/tests/tests.sh

```diff
@@ -339,4 +339,39 @@ fi
 
 rm -r test-download
 
+# Download encrypted file by using the sda download service
+# Create a user key pair
+if ( echo "" | ./sda-cli createKey user_key ) ; then
+    echo "Created a user key pair for downloading encrypted files"
+else
+    echo "Failed to create a user key pair for downloading encrypted files"
+    exit 1
+fi
```

Comment:
Thanks for the suggestion. I still think it is better to have some error handling. However, I have modified `echo ""` to `"yes ""` to generate continuous stream of empty inputs. see the commit 85e4691

Reply:
Apart from that, I have also renamed the flag `-public-key` to `-pubkey` so that it is in consistent with the `htsget` sub-command.

Reply:
It is far from a deal breaker for me but since this is an integration test I think we don't need to be too formal about it. In addition, if any error occurs the test will fail and exit either way since `set -e` is included at the top of the script.